src/main/java/frc/robot/subsystems/turret/commands/VisionTurret.java

```diff
@@ -0,0 +1,53 @@
+package frc.robot.subsystems.turret.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Constants;
+import frc.robot.subsystems.turret.Turret;
+import frc.robot.utilities.VisionModule;
+
+public class VisionTurret extends CommandBase {
+
+    private Turret turret;
+    private VisionModule visionModule;
+
+    public VisionTurret(Turret turret, VisionModule visionModule) {
+        this.turret = turret;
+        addRequirements(turret);
+        this.visionModule = visionModule;
+    }
+
+    public boolean targetSeen() {
+        return visionModule.targetSeen();
+    }
+
+    public double getVisionAngle() {
+        return visionModule.getVisionAngle() + turret.getPosition();
+    }
+
+    public void setLEDs(boolean on) {
+        visionModule.setLEDs(on);
+    }
+
+    @Override
+    public void initialize() {
+        setLEDs(true);
+    }
+
+    @Override
+    public void execute() {
+        if (targetSeen())
+            turret.setAngle(getVisionAngle());
```

Comment:
Add current angle.

Reply:
What? @Adam14boat I think you should look at the function (he created this function in this command)

Reply:
You are right I didn't see that

Reply:
You're right I didn't see that